_config.yml

```diff
@@ -12,3 +12,16 @@ sass:
   style: compressed
   load_paths:
     - node_modules/
+
+plugins:
+  - jekyll-seo-tag
+
+title: "👋  Hello, GitHub"
```

Comment:
It seems there is an extra space in the title.

Reply:
Sorry about that. Fixed via 414640a.

Reply:
👍